JanePhotoBrowser/PhotoBrowser/PhotoBrowserFullscreen/PhotoBrowserFullscreenViewController.swift

```diff
@@ -88,8 +108,13 @@ public class PhotoBrowserFullscreenViewController: UIViewController {
         }
         
         self.closeButton.isAccessibilityElement = false
-        self.closeButton.image = PhotoBrowserIconography.imageOfXIcon()
+        if #available(iOS 13, *) {
+            self.closeButton.image = PhotoBrowserIconography.imageOfXIcon()
+        } else {
+            PhotoBrowserIconography.drawXIcon()
```

Comment:
This won't work

Reply:
Why won't it?

Reply:
Because you aren't setting anything to the button, you are just drawing an image

Reply:
Oh never mind, I didn't realize I was calling the wrong function. This is why we need PRs 😅